pkg/api/defaults-kubelet_test.go

```diff
@@ -45,7 +45,7 @@ func TestKubeletConfigDefaults(t *testing.T) {
 		"--max-pods":                          strconv.Itoa(DefaultKubernetesMaxPods),
 		"--network-plugin":                    NetworkPluginKubenet,
 		"--node-status-update-frequency":      K8sComponentsByVersionMap[cs.Properties.OrchestratorProfile.OrchestratorVersion]["nodestatusfreq"],
-		"--non-masquerade-cidr":               DefaultNonMasqueradeCIDR,
+		"--non-masquerade-cidr":               "10.240.0.0/12",
```

Comment:
This is now the proper expected value for not having any "addons" as part of the input cs object

Reply:
can you explain why that's related?

Reply:
And why not continue to use a constant here?